actions/class.Runner.php

```diff
@@ -259,12 +269,11 @@ protected function getErrorCode($e = null)
      */
     public function init()
     {
-        /** @var QtiRunnerServiceContext $serviceContext */
-        $serviceContext = $this->getRunnerService()->initServiceContext($this->getServiceContext());
-
         $this->checkSecurityToken();
 
         try {
+            /** @var QtiRunnerServiceContext $serviceContext */
+            $serviceContext = $this->getRunnerService()->initServiceContext($this->getServiceContext());
             $this->returnJson($this->getInitResponse($serviceContext));
         } catch (Exception $e) {
             $this->returnJson(
```

Comment:
As it is general Exception - could you at least store Log::e($e->getMessage())?

Reply:
agree

Reply:
I also added logs for all exceptions where it was missed